docs/SmartStore.md

```diff
@@ -242,3 +242,16 @@ See [indexes.conf](https://docs.splunk.com/Documentation/Splunk/latest/Admin/Ind
 | maxCacheSize | max_cache_size  | [cachemanager] |
 | evictionPolicy |eviction_policy  |[cachemanager] |
 | evictionPadding | eviction_padding  |[cachemanager] |
+
+## Additional configuration
+
+If there is a need to configure Smartstore or Index related configs in addition to the above, this can be achieved by including the additional configs via Apps:
```

Comment:
This is what I've got in my mind:

There are SmartStore settings that are not covered by the SmartStore CR. 
If you need to configure SmartStore or other index related settings outside of what's available in the CR, this can be achieved by using an app.

Reply:
I have updated accordingly, thanks @eddikay